core/src/main/java/org/apache/accumulo/core/iterators/user/IntersectingIterator.java

```diff
@@ -96,13 +96,6 @@ public static class TermSource {
     public Collection<ByteSequence> seekColfams;
     public final boolean notFlag;
 
-    public TermSource(TermSource other) {
-      this.iter = other.iter;
-      this.term = other.term;
-      this.notFlag = other.notFlag;
-      this.seekColfams = other.seekColfams;
-    }
-
```

Comment:
I think this is public API. Maybe users have a utility for this?

Reply:
Reverted changes to this class in 700ec8c